test/spec/utils.js

```diff
@@ -0,0 +1,16 @@
+exports.getSearchCount = function*() {
+  var value = yield browser
+    .pause(500)
+    .waitForText('.results-main-alert .u-group_inline-left h1 span:first-child', 10000)
+    .getText('.results-main-alert .u-group_inline-left h1 span:first-child');
+  return parseInt(value.replace(/,/g, ""), 10);
+};
+
+exports.runSearch = function*(ops) {
+  yield browser
```

Comment:
'yield' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'browser' is not defined.